misc/diagram-util/src/main/java/org/ballerinalang/diagramutil/SyntaxTreeMapGenerator.java

```diff
@@ -251,7 +251,9 @@ private JsonObject visibleEP(Node node, TypeSymbol typeSymbol, boolean isRemoteA
         JsonObject symbolMetaInfo = new JsonObject();
         ModuleID moduleID = typeSymbol.getModule().isPresent() ? typeSymbol.getModule().get().id() : null;
         String orgName = moduleID != null ? moduleID.orgName() : "";
+        String packageName = moduleID != null ? moduleID.packageName() : "";
         String moduleName = moduleID != null ? moduleID.moduleName() : "";
+        String version = moduleID != null ? moduleID.version() : "";
 
         if (node.kind() == SyntaxKind.REQUIRED_PARAM) {
```

Comment:
If we consider the if-else ladder we have used here, there is a considerable amount of duplication for setting the properties such as `typeName`, `orgName` and etc. Let's refactor this one in this PR itself. Check whether if we can come up with a solution based on a design pattern as well (Builder pattern maybe).

Reply:
sure. will refactor this.